rules/okta_rules/okta_group_admin_role_assigned.yml

```diff
@@ -0,0 +1,156 @@
+AnalysisType: rule
+Description: Detect when an admin role is assigned to a group
+DisplayName: Okta Group Admin Role Assigned
+Enabled: true
+Filename: okta_group_admin_role_assigned.py
```

Comment:
I think of this as [T1548](https://attack.mitre.org/techniques/T1548/), though att&ck suggests that is specific to operating systems.